lib/chromedriver.js

```diff
@@ -16,11 +16,11 @@ const DEFAULT_HOST = '127.0.0.1';
 const DEFAULT_PORT = 9515;
 class Chromedriver extends events.EventEmitter {
   constructor (args = {}) {
-    const {host, port, executable, cmdArgs, adbPort, verbose, logPath} = args;
+    const {host, port, executable, cmdArgs, adb, verbose, logPath} = args;
     super();
     this.proxyHost = host || DEFAULT_HOST;
     this.proxyPort = port || DEFAULT_PORT;
-    this.adbPort = adbPort;
+    this.adb = adb;
```

Comment:
we should remember that this will be a breaking change, so we should publish this as a new major version of the package